### PR TITLE
Rules: Command to move a replication rule; Closes #133

### LINKED ISSUE #133: Option to "move" a rule in CLI
Motivation
----------
The idea is basically the same principle as with BB8. A specific rule should be moved to another RSE, including all it's properties.

_Migrated from [JIRA](https://its.cern.ch/jira/browse/RUCIO-2721)_

### PR DESCRIPTION
**Pull request by bari12**
Rules: Command to move a replication rule; Closes #133

### CODE REVIEW

File: bin/rucio [hunk 1]
```diff
@@ -2139,6 +2151,7 @@ def info_rule(args):
         print "Purge replicas:             %s" % rule['purge_replicas']
         print "Notification:               %s" % rule['notification']
         print "End of life:                %s" % rule['eol_at']
+        print "Child rule Id:              %s" % rule['child_rule_id']
```
Comment (mlassnig) [inline bin/rucio]: capital R in rule or lowercase I in id just to be consistent ;-)

**Comment by mlassnig** [comment]
look straightforward to me, any implications?

File: bin/rucio (hunk 1 above)
Comment (bari12) [reply bin/rucio]: 🙄  will fix it 😄

**Comment by bari12** [comment]
No it should be fine, I did not test it massively, but will ask site admins to test it first a bit before large scale use.

**Pull request closed without merging**